python/seldon_core/seldon_methods.py

```diff
@@ -177,7 +177,6 @@ def send_feedback(
     -------
 
     """
-    is_proto = isinstance(request, prediction_pb2.SeldonMessage)
```

Comment:
Ok this is actualyl a bug. It should be `prediction_pb2.Feedback` instead... could you  try that instead? Happy to add the fix, and rerun the tests, not sure how this wasnt' caught before, we should probably extend coverage of tests for that